apps/accounts/management/commands/model2csv.py

```diff
@@ -74,6 +82,9 @@ def add_arguments(self, parser):
                                                      " and export time as "
                                                      "columns: True | False")
 
+        parser.add_argument('--filter_fields', help="filter fields by column "
```

Comment:
Please consider using the actual field names here. This will go a long way towards helping our future selves understand what exactly this command is doing after we've forgotten what fields the column numbers map to. 😉

Reply:
Good idea. Not that hard to do so I am submitting updates for both web-server and web-deployment.

Reply:
@ekivemark status of this PR?